src/components/DocLayout/DocLayout.scss

```diff
@@ -14,6 +14,7 @@
 
     &__left,
     &__right {
+        order: 3;
```

Comment:
Why is adding "order" property necessary ?

Reply:
Changing order in html affects on focus order. But it also affects on Minitoc position in UI. Minitoc should be in the right side of layout and main content should be in the middle (Toc -> Main content -> MiniToc). Without "order" MiniToc will be in the middle of layout and main content will be in the right side (Toc -> MiniToc -> Main content).